client/src/utils/tone/index.ts

```diff
@@ -0,0 +1,83 @@
+import store from 'store';
+import { FlashMessageArg } from '../../components/Flash/redux';
+import { Themes } from '../../components/settings/theme';
+
+// TODO: Add more tone types
+export async function playTone(
+  state: FlashMessageArg['message'] | Themes | string
+): Promise<void> {
+  const playSound = !!store.get('fcc-sound');
+  if (playSound) {
+    const tone = await import('tone');
+    if (tone.context.state !== 'running') {
+      void tone.context.resume();
```

Comment:
```suggestion
      tone.context.resume();
```
What does 'void' do here?

Reply:
I'm gonna guess Shaun did some copy-pasting from the old logic to this logic - I added the `void` in places where I couldn't `await` without rewriting the whole file. `void` tells TypeScript that I'm not doing anything with the promise result, so it doesn't need to be awaited. (linter gets mad otherwise).

Reply:
Ah, so it does.  How about this?
```suggestion
      tone.context.resume().catch(err => {
        console.error('Error resuming audio context', err);
      });
```

Reply:
Are we happy showing the error in prod?

Reply:
We already are, aren't we?  In general, I don't think it's terrible to do so. At least until we get Sentry set up.

Reply:
> At least until we get Sentry set up.

Is that still a thing?!

Reply:
https://github.com/freeCodeCamp/freeCodeCamp/pull/43920 :cold_sweat: